src/Controller/IndexController.php

```diff
@@ -91,7 +101,7 @@ public function init()
         $viewParams['connectorPath'] = $connectorPath;
 
         $fileType = $this->getEvent()->getRouteMatch()->getParam('fileType');
-        if (!empty($fileType)) {
+        if (!empty($type)) {
```

Comment:
Sorry I' don't know why this part of code change, I pull a new version with the correct code.